jest.setup.ts

```diff
@@ -8,4 +8,6 @@ expect.extend({
   ...jestMatchers,
 });
 
+expect.extend({ toBeDeepCloseTo, toMatchCloseTo });
```

Comment:
You can just update the existing extend above.

Reply:
Woops '^^